data-binning/source/Properties/launchSettings.json

```diff
@@ -3,7 +3,9 @@
     "Cosmos_Patterns_Bucketing": {
```

Comment:
I think you can change Cosmos_Patterns_Bucketing to "data_binning" to match the project name.